tests/phpunit/tests/admin/includesPlugin.php

```diff
@@ -33,6 +33,36 @@ public function test_get_plugin_data() {
 			$this->assertArrayHasKey( $name, $data );
 			$this->assertSame( $value, $data[ $name ] );
 		}
+
+		$data = get_plugin_data( DIR_TESTDATA . '/plugins/requirescp.php' );
+
+		$default_headers = array(
+			'Name'        => 'With requires CP and without Update URI',
+			'RequiresCP'  => '1.5',
+			'UpdateURI'   => 'https://directory.classicpress.net/wp-json/wp/v2/plugins?byslug=plugins',
+		);
+
+		$this->assertTrue( is_array( $data ) );
+
+		foreach ( $default_headers as $name => $value ) {
+			$this->assertTrue( isset( $data[ $name ] ) );
+			$this->assertSame( $value, $data[ $name ] );
+		}
+
+		$data = get_plugin_data( DIR_TESTDATA . '/plugins/updateuri.php' );
+
+		$default_headers = array(
+			'Name'        => 'With requires CP and with Update URI',
+			'RequiresCP'  => '1.5',
+			'UpdateURI'   => 'https://directory.classicpress.net/wp-json/wp/v2/plugins?byslug=keep-it-safe',
```

Comment:
This link 404s on the server, can it be made to show a test if anyone directly calls it?